tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -598,7 +603,14 @@ static void stream_released(struct bt_bap_stream *stream)
 		cigs[stream->ep->cig_id] = NULL;
 	}
 
-	a_stream->ase_id = 0;
+	stream_free(a_stream);
+	a_stream = NULL;
+
+	if (stream->ep->dir == BT_AUDIO_DIR_SINK) {
```

Comment:
You memset the stream in line 606 and then access `stream->ep` when `a_stream` seems to be the container of `stream`. Doesn't this also memset `stream` and thus causing NULL pointer access?

Reply:
Ah, yes, fixed now